### PR TITLE
Fix for GCW-2904 (part 2)

### PR DESCRIPTION
**Pull request by patrixr**
More backlink fixes for ticket GCW-2904:

Change: Switch from `transitionTo` to `history.back()`

### CODE REVIEW

File: app/components/cancel-order.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import Ember from "ember";
-import AjaxPromise from 'stock/utils/ajax-promise';
+import AjaxPromise from "stock/utils/ajax-promise";
 const { getOwner } = Ember;
 
 export default Ember.Component.extend({
-  tagName: 'a',
+  tagName: "a",
```
Comment (steveyken) [inline app/components/cancel-order.js]: Mission critical change. :100:

File: app/helpers/back-action.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default Ember.Helper.extend({
 
       const noHistory = window.history.length === 1;
       const deepLinked =
-        document.referrer && document.referrer.indexOf(config.APP.NAME) < 0;
+        document.referrer && document.referrer.indexOf(config.APP.ORIGIN) < 0;
```
Comment (abulsayyad123) [inline app/helpers/back-action.js]: What does `document.referrer` returns ?

Comment (patrixr) [reply app/helpers/back-action.js]: It's just a string with the url of the page that redirected to your app.

So if on "jira.com" I click on a `stock.goodcity.hk` link, then referrer will be Jira

File: app/components/cancel-order.js (hunk 1 above)
Comment (patrixr) [reply app/components/cancel-order.js]: That's the prettifier hook :'(

File: app/helpers/back-action.js (hunk 2 above)
Comment (patrixr) [reply app/helpers/back-action.js]: so if the referrer is Stock, that means I navigated from my app -> to my app

Comment (abulsayyad123) [reply app/helpers/back-action.js]: Thanks 👍  for clarifying.

**Pull request merged**